src/main/io/osd.c

```diff
@@ -547,25 +557,25 @@ void osdFormatAltitudeSymbol(char *buff, int32_t alt)
         case OSD_UNIT_IMPERIAL:
             if (osdFormatCentiNumber(buff + 4 - digits, CENTIMETERS_TO_CENTIFEET(alt), 1000, 0, 2, digits)) {
                 // Scaled to kft
-                buff[4] = SYM_ALT_KFT;
+                buff[symbolIndex++] = symbolKFt;
```

Comment:
Could this KFt formatting be removed completely? I think most people using imperial units would prefer to see altitude always in feet without decimals. Ie. 1500 ft. vs 1.500.

Reply:
No, due to space. There is only room for 3 digits. INAV has been kFt for years.